changelogs/fragments/redhat_subscribtion.yml

```
@@ -0,0 +1,5 @@
+bugfixes:
+  - made code compatible with Python3
+minor_changes:
+  - do not call subscribtion-manager command, when it is not necessary
```

These fragments are combined into a restructured text file, so you can use those formatting shortcuts for things like inline code.
```suggestion
  - redhat_subscription - do not call ``subscribtion_manager`` command when it is not necessary
```

---

Done